dowhy/causal_estimator.py

```diff
@@ -225,6 +227,34 @@ def estimate_effect_naive(self):
         est = np.mean(df_withtreatment[self._outcome_name]) - np.mean(df_notreatment[self._outcome_name])
         return CausalEstimate(est, None, None, control_value=0, treatment_value=1)
 
+    def effect_tt(self, df: pd.DataFrame, *args, **kwargs):
+        """
+        Effect of treatment on the treated
```

Comment:
we use sphinx style `:param x:` docs.

Reply:
No longer relevant as I now removed the method